src/libs/actions/PersonalDetails.js

```diff
@@ -69,8 +69,12 @@ function getDisplayName(login, personalDetail) {
 
     const firstName = userDetails.firstName || '';
     const lastName = userDetails.lastName || '';
+    const fullName = (`${firstName} ${lastName}`).trim();
+    if (fullName) {
+        return fullName;
+    }
 
-    return (`${firstName} ${lastName}`).trim() || userLogin;
+    return userDetails.displayName || userLogin;
```

Comment:
do we need `userDetails.displayName` at this point? If they don't have a first or last name, but they do have a displayName, then it would seem something is out of sync.

Could we instead: 
```JS
return fullName || userLogin;
```